plugins/chat/assets/javascripts/discourse/services/chat-subscriptions-manager.js

```diff
@@ -205,6 +206,8 @@ export default class ChatSubscriptionsManager extends Service {
             channel.tracking.unreadCount++;
           }
 
+          this.chatChannelNotificationSound.play(channel);
```

Comment:
this specific line is not tested as it's hard to test an actual sound is played or even this function is called in this case, however the underlying service is fully tested